lib/command/run-multiple.js

```diff
@@ -71,28 +84,41 @@ module.exports = function (suites, options) {
         return b === browser;
       });
     }
-
     // throw error if no configuration
     if (suiteConf.browsers.length === 0) throw new Error(`Browser ${browser} not found in multiple suite "${suiteName}" config`);
 
-    runSuite(suiteName, suiteConf);
+    totalSubprocessCount += suiteConf.browsers.length;
```

Comment:
looks like here this count is unnecessary. You reassign is later `totalSubprocessCount = forksToExecute.length;`

Reply:
yep, I'll remove it now. Oversight on my part.